protocol/x/listing/keeper/msg_server_enable_pml.go

```diff
@@ -0,0 +1,34 @@
+package keeper
+
+import (
+	"context"
+
+	errorsmod "cosmossdk.io/errors"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/dydxprotocol/v4-chain/protocol/lib"
+	"github.com/dydxprotocol/v4-chain/protocol/x/listing/types"
+)
+
+func (k msgServer) EnablePermissionlessMarketListing(
```

Comment:
similar to above comment about renaming to `MsgSetEnable...`, wonder if this should be renamed accordingly